config/config_test.go

```diff
@@ -67,6 +67,46 @@ func TestBlacklist(t *testing.T) {
 	}
 }
 
+func TestArgsBlacklist(t *testing.T) {
```

Comment:
Can we also have a benchmark added for blacklisting by regex? I'm curious to know how these regexes will perform.

This resource should be able to help:
https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go